CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
maybe we could try to import this from llvm cmake project to be use to be in sync